lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -181,6 +185,18 @@ final class AddComicViewController: UIViewController, UIPickerViewDelegate, UIPi
         deregisterFromKeyboardNotifications()
     }
 
+    @IBAction func addCoverButtonTapped(_ sender: UIButton) {
+        if AVCaptureDevice.authorizationStatus(for: AVMediaType.video) ==  AVAuthorizationStatus.authorized {
```

Comment:
All of the enum types (`AVMediaType`, `AVAuthorizationStatus`) can probably be inferred by the compiler and aren't necessary